packages/web/components/ui/block-suggestion.tsx

```
@@ -133,38 +132,38 @@ export function BlockSuggestionCard({
               {formatCommentDate(new Date(suggestion.createdAt))}
             </span>
           </div>
         </div>
 
-        <div className="relative mt-1 mb-4 pl-[32px]">
-          <div className="flex flex-col gap-2">
+        <div className='relative mt-1 mb-4 pl-[32px]'>
+          <div className='flex flex-col gap-2'>
             {suggestion.type === 'remove' && (
               <React.Fragment>
                 {suggestionText2Array(suggestion.text!).map((text, index) => (
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)